core/commands/object.go

```diff
@@ -344,8 +343,8 @@ Data should be in the format specified by the --inputenc flag.
 }
 
 // objectData takes a key string and writes out the raw bytes of that node (if there is one)
-func objectData(n *core.IpfsNode, fpath path.Path) (io.Reader, error) {
-	dagnode, err := n.Resolver.ResolvePath(fpath)
+func objectData(n *core.IpfsNode, fpath string) (io.Reader, error) {
```

Comment:
we want to use `path.Path` wherever possible. it lends additional type safety and lets us know what this type of thing is. ultimately, a `path.Path` is just a string, so can just cast it to a string if need be.